src/graphics/aurora/text.cpp

```diff
@@ -242,6 +243,10 @@ void Text::parseColors(const Common::UString &str, Common::UString &parsed,
 	}
 }
 
+void Text::setFont(Common::UString fnt) {
```

Comment:
You're not changing the string inside the method, so it can be a const reference, i.e. `const Common::UString &`.